tests/installation/disable_grub_timeout.pm

```diff
@@ -47,7 +47,8 @@ sub run {
     }
 
     # Config bootloader is not be supported during an upgrade
-    if (get_var('UPGRADE') && (!is_sle('<15') || !is_leap('<15.0'))) {
+    # Add exception for SLES11SP4 base update, configure grub for this scenario
```

Comment:
Can you state how sle11 is different? I.e. refer to grub1 being upgraded to grub2

Reply:
refer to bsc#1070233 https://bugzilla.suse.com/show_bug.cgi?id=1070233#c14
SLES11 use grub1, need configure Gurb2 for migration here.

Reply:
Yes, of course. But I mean this could be stated in the file. Nevermind :)